test/addons/rook-cluster/fetch

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+from drenv import cache
+
+os.chdir(os.path.dirname(__file__))
+path = cache.path("addons/cluster-test.yaml")
```

Comment:
This is the cache of the rook-cluster addon, so we should name it "addons/rook-cluster.yaml"